lib/portlayer/network/ipam.go

```diff
@@ -281,13 +283,25 @@ func (s *AddressSpace) ReserveIP4Range(firstIP net.IP, lastIP net.IP) (*AddressS
 		}
 
 		// found range
+		log.Infof("Reserving IP range [%s, %s]", firstIP.String(), lastIP.String())
```

Comment:
Should also be using op trace as there's no way to correlate at all - don't even have container ID in here.

Reply:
In order for op to be useful here, I would need to weave op throughout the code path.  Given the time left, I didn't do it.